renderer/hooks/useOnClickOutside.js

```diff
@@ -1,16 +1,33 @@
 import { useEffect } from 'react'
 
 /**
- * React hook that calls a handler when a clik event comes from outside of a ref.
+ * isInside - Check if an event comes from a nested element.
+ *
+ * @param  {object} ref Element reference
+ * @returns {boolean} Boolean indicating wether event originates from nested element
  */
-export default function useOnClickOutside(ref, handler) {
+function isInside(ref) {
+  if (!ref.current || ref.current.contains(this.target)) {
```

Comment:
could be just `return !ref.current || ref.current.contains(this.target)`

Reply:
Updated